src/main/java/com/cloudogu/wiki/RefreshWikisServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.cloudogu.wiki;
+
+import com.cloudogu.wiki.scmm.SessionCacheScmWikiListStrategy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * @author Maren Süwer
+ */
+
+public class RefreshWikisServlet extends HttpServlet {
+
+    private static final Logger LOG = LoggerFactory.getLogger(RefreshWikisServlet.class);
+    private final String refreshUrl;
+
+    public RefreshWikisServlet() {
+        refreshUrl = "/smeagol";
```

Comment:
Do not use a hardcoded url, please use the [context path from the request](https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletRequest.html#getContextPath()).